lib/exceptions.js

```diff
@@ -0,0 +1,54 @@
+/*jshint es5:true */
```

Comment:
`'use strict';` here?
(Or is it already enabled by default in the versions of Node.js we're using?)